cmake/find_fastops.cmake

```diff
@@ -1,4 +1,6 @@
-option (ENABLE_FASTOPS "Enable fast vectorized mathematical functions library by Mikhail Parakhin" ${NOT_UNBUNDLED})
+if (NOT ARCH_ARM AND NOT OS_FREEBSD)
```

Comment:
What's wrong with FreeBSD?

Reply:
`[18/1265] Building CXX object contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx/ops_avx.cpp.o
FAILED: contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx/ops_avx.cpp.o 
/usr/local/bin/ccache /usr/local/bin/clang++80  -DHAVE_STDINT_H -isystem ../contrib/fastops -fdiagnostics-color=always -fsized-deallocation  -gdwarf-aranges -pipe -msse4.1 -msse4.2 -mpopcnt  -fno-omit-frame-pointer  -Wall -Wno-unused-command-line-argument  -Wnon-virtual-dtor -w -std=c++1z -O2 -g -DNDEBUG -O3    -std=c++17 -mavx -DNO_AVX2 -MD -MT contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx/ops_avx.cpp.o -MF contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx/ops_avx.cpp.o.d -o contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx/ops_avx.cpp.o -c ../contrib/fastops/fastops/avx/ops_avx.cpp
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:715:38: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set1(int64_t v) {
                                     ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:718:37: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2) {
                                    ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:718:49: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2) {
                                                ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:850:45: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i LoadU(const int64_t* p) {
                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:853:44: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Load(const int64_t* p) {
                                           ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:862:41: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void StoreU(int64_t* p, t_i v) {
                                        ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:865:40: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void Store(int64_t* p, t_i v) {
                                       ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:928:45: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i LoadU(const int64_t* p) {
                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:931:44: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Load(const int64_t* p) {
                                           ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:940:41: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void StoreU(int64_t* p, t_i v) {
                                        ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:943:40: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void Store(int64_t* p, t_i v) {
                                       ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:971:45: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i LoadU(const int64_t* p) {
                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:974:44: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Load(const int64_t* p) {
                                           ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:983:41: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void StoreU(int64_t* p, t_i v) {
                                        ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:986:40: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void Store(int64_t* p, t_i v) {
                                       ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1001:38: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set1(int64_t v) {
                                     ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1007:37: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2, int64_t v3, int64_t v4) {
                                    ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1007:49: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2, int64_t v3, int64_t v4) {
                                                ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx/ops_avx.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1007:61: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2, int64_t v3, int64_t v4) {
                                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
[23/1265] Building CXX object contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx2/ops_avx2.cpp.o
FAILED: contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx2/ops_avx2.cpp.o 
/usr/local/bin/ccache /usr/local/bin/clang++80  -DHAVE_STDINT_H -isystem ../contrib/fastops -fdiagnostics-color=always -fsized-deallocation  -gdwarf-aranges -pipe -msse4.1 -msse4.2 -mpopcnt  -fno-omit-frame-pointer  -Wall -Wno-unused-command-line-argument  -Wnon-virtual-dtor -w -std=c++1z -O2 -g -DNDEBUG -O3    -std=c++17 -mavx2 -mfma -MD -MT contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx2/ops_avx2.cpp.o -MF contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx2/ops_avx2.cpp.o.d -o contrib/fastops-cmake/CMakeFiles/fastops.dir/__/fastops/fastops/avx2/ops_avx2.cpp.o -c ../contrib/fastops/fastops/avx2/ops_avx2.cpp
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:715:38: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set1(int64_t v) {
                                     ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:718:37: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2) {
                                    ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:718:49: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2) {
                                                ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:850:45: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i LoadU(const int64_t* p) {
                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:853:44: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Load(const int64_t* p) {
                                           ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:862:41: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void StoreU(int64_t* p, t_i v) {
                                        ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:865:40: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void Store(int64_t* p, t_i v) {
                                       ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:928:45: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i LoadU(const int64_t* p) {
                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:931:44: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Load(const int64_t* p) {
                                           ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:940:41: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void StoreU(int64_t* p, t_i v) {
                                        ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:943:40: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void Store(int64_t* p, t_i v) {
                                       ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:971:45: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i LoadU(const int64_t* p) {
                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:974:44: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Load(const int64_t* p) {
                                           ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:983:41: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void StoreU(int64_t* p, t_i v) {
                                        ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:986:40: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static void Store(int64_t* p, t_i v) {
                                       ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1001:38: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set1(int64_t v) {
                                     ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1007:37: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2, int64_t v3, int64_t v4) {
                                    ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1007:49: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2, int64_t v3, int64_t v4) {
                                                ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
In file included from ../contrib/fastops/fastops/avx2/ops_avx2.cpp:3:
In file included from ../contrib/fastops/fastops/core/FastIntrinsics.h:6:
../contrib/fastops/fastops/core/SIMDFunctions.h:1007:61: error: unknown type name 'int64_t'; did you mean 'uint64_t'?
        FORCE_INLINE static t_i Set(int64_t v1, int64_t v2, int64_t v3, int64_t v4) {
                                                            ^
/usr/include/sys/endian.h:52:20: note: 'uint64_t' declared here
typedef __uint64_t      uint64_t;
                        ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
`